data/migrations/20190424165604_create_tables.js

```diff
@@ -15,9 +15,144 @@ exports.up = function(knex) {
         tbl.string('stripe_product_id').notNullable();
         tbl.string('stripe_plan_id').notNullable();
       })
+      //Users table
+      .createTable('users', tbl => {
+        tbl.increments();
+        tbl
+          .integer('tier_id')
+          .unsigned()
+          .references('id')
+          .inTable('tiers')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl
+          .string('email', 128)
+          .notNullable()
+          .unique();
+        tbl.string('logo_url', 400);
+        tbl.string('stripe_customer_id', 400);
+      })
+      //Categories Table
+      .createTable('categories', tbl => {
+        tbl.increments();
+        tbl
+          .integer('user_id')
+          .unsigned()
+          .references('id')
+          .inTable('users')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl.string('name', 128);
+        tbl.timestamp('timestamps').defaultTo(knex.fn.now());
+      })
+      //Games Table
+      .createTable('games', tbl => {
+        tbl.increments();
+        tbl.string('name', 128).notNullable();
+        tbl.timestamp('timestamps').defaultTo(knex.fn.now());
+        tbl.string('last_played', 128).notNullable();
+        tbl
+          .integer('user_id')
+          .unsigned()
+          .references('id')
+          .inTable('users')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl.string('logo_url', 400);
+      })
+      //Rounds table
+      .createTable('rounds', tbl => {
+        tbl.increments();
+        tbl
+          .integer('game_id')
+          .unsigned()
+          .references('id')
+          .inTable('games')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl.timestamp('timestamps').defaultTo(knex.fn.now());
+        tbl.integer('number').notNullable();
+      })
+      //Question types table
+      .createTable('question_types', tbl => {
+        tbl.increments();
+        tbl
+          .integer('user_id')
+          .unsigned()
+          .references('id')
+          .inTable('users')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl.string('name', 128);
+        tbl.timestamp('timestamps').defaultTo(knex.fn.now());
+      })
+
+      //Questions table
+      .createTable('questions', tbl => {
+        tbl.increments();
+        tbl
+          .integer('user_id')
+          .unsigned()
+          .references('id')
+          .inTable('users')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl
+          .integer('question_type_id')
+          .unsigned()
+          .references('id')
+          .inTable('question_types')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
+        tbl
+          .integer('category_id')
+          .unsigned()
+          .references('id')
+          .inTable('categories')
+          .onDelete('CASCADE')
+          .onUpdate('CASCADE');
```

Comment:
How is this going to work? Cuz the category IDs we send will need to match the API in order to fetch the right questions.

![image](https://user-images.githubusercontent.com/8473052/56826105-4873b080-6829-11e9-9a84-8374cf97e307.png)

We could add another entry to categories like `trivia_id` or some such.